waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -522,6 +537,185 @@ protected String getUsage() {
         }
     }
 
+    /**
+     * Use {@code Producer} command to create a producer client that
+     * submits transactions to Waltz server.
+     *
+     * Producer process is closed once total sum of high watermarks for active
+     * partitions reaches num-callbacks.
+     */
+    private static final class Producer extends ClientCliBaseClass {
```

Comment:
Missing implementation of create-producers and create-consumers unit test in ClientCliTest.